api/lambdas/src/fitbit.ts

```diff
@@ -0,0 +1,70 @@
+import * as Sentry from "@sentry/serverless";
+import { Request } from "express";
+import { getSecret } from "@aws-lambda-powertools/parameters/secrets";
+import { createHmac } from "crypto";
+import axios from "axios";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const apiServerURL = getEnvOrFail("API_URL");
+
+const api = axios.create();
+
+const buildResponse = (status: number, body?: unknown) => ({
+  statusCode: status,
+  body,
+});
+
+const defaultResponse = () => buildResponse(200);
+
+export const handler = Sentry.AWSLambda.wrapHandler(async (req: Request) => {
+  console.log(`Verifying at least one UserAuthToken on body...`);
+
+  if (!req.body) {
+    console.log("Resquest has no body - will not be forwarded to the API");
+    return defaultResponse();
+  }
+
+  const secret = await getSecret("FITBIT_CLIENT_SECRET");
```

Comment:
you're not using the env var here - which negates the purpose of using the env var in the first place, there's a reason the example I gave you does this:

```
const sidechainFHIRConverterKeysSecretName = isSidechainConnector()
  ? getEnvOrFail("SIDECHAIN_FHIR_CONVERTER_KEYS")
  : undefined;
  ```

Reply:
Not sure I understand this part. That example from `sqs-to-converter.ts` is very different with a bunch of other env variables involved..

Reply:
The idea is the same - it's reading an env var which contains the name of the secret for which to read the value from.

In your current code, you're doing `getSecret("FITBIT_CLIENT_SECRET");`. This assumes that the secret in AWS is called `FITBIT_CLIENT_SECRET`, which isn't true since in the CDK code you're setting the env var like so `FITBIT_CLIENT_SECRET: fitbitClientSecret,`, and `fitbitClientSecret` could be [set to any value](https://github.com/metriport/metriport-internal/blob/53bfae34f60d3172e8c08fe1b6ac64372818ea1b/config-oss/api-infra/production.ts#L26).

So, as per the example, you'll want to do something like:

```
const fitbitClientSecretName = getEnvOrFail("FITBIT_CLIENT_SECRET");
....
const secret = await getSecret(fitbitClientSecretName);
```

Lmk if this makes sense